src/dataSources/cloudFirestore/session.js

```diff
@@ -25,6 +26,34 @@ const session = dbInstance => {
     return results;
   }
 
+  async function findAllApprovedByEventIdAtDate(eventId, atDate, daysAfter) {
+    dlog(
+      'findAllApprovedByEventIdAtDate(eventId, atDate, daysAfter)',
+      eventId,
+      atDate,
+      daysAfter,
+    );
+    let query = sessionsCollections
+      .where('eventId', '==', eventId)
+      .where('status', 'in', approvedSessionStatuses);
+
+    if (atDate) {
+      const fromdate = new Date(atDate);
+      query = query.where('startTime', '>=', atDate);
```

Comment:
should this be using fromdate (and it should be fromDate) and not atDate?

Reply:
Oops, good find.. Refactoring error.

Reply:
@csell5 corrected with last commit 5445cc9